overlord/auth/auth.go

```diff
@@ -206,6 +208,8 @@ NextUser:
 
 // An AuthContext handles user updates.
 type AuthContext interface {
```

Comment:
While we don't have a strong policy about this I always love to see documentation about interface methods.